.changeset/every-apes-return.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/table': patch
```

Comment:
Since we're adding the new testid, this should be `minor`. `Patch` is for fixes, while `minor` is for additions.